src/bitmap/store/array_store/mod.rs

```diff
@@ -106,6 +106,15 @@ impl ArrayStore {
         (pos_end - pos_start) as u64
     }
 
+    pub fn remove_first(&mut self, n: usize) {
+        self.vec.rotate_left(n.into());
+        self.vec.truncate(self.vec.len() - n as usize);
```

Comment:
Can't we use [the `Vec::drain` method](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.drain) here, too?

Reply:
fixed
https://github.com/RoaringBitmap/roaring-rs/pull/253/commits/5761b1c9bc2e648be5a5d425242d808d5b266984